pychromecast/controllers/plex.py

```diff
@@ -1,40 +1,538 @@
+        self._send_cmd({MESSAGE_TYPE: TYPE_PLAY})
+
+    def previous(self):
+        """Send previous command."""
+        self._send_cmd({MESSAGE_TYPE: TYPE_PREVIOUS})
+
+    def next(self):
+        """Send next command."""
+        self._send_cmd({MESSAGE_TYPE: TYPE_NEXT})
+
+    def seek(self, position, resume_state="PLAYBACK_START"):
+        """Send seek command
+
+        Args:
+            position (int): offset in seconds.
+            resume_state (str, default): PLAYBACK_START
+        """
+        self._send_cmd(
+            {MESSAGE_TYPE: TYPE_SEEK,
+             SEEK_KEY: position,
+             "resumeState": resume_state}
+        )
+
+    def rewind(self):
+        """Rewind back to the start"""
+        self.seek(0)
+
+    def set_volume(self, percent):
+        """Set the volume 1-100
+
+        Args:
+            percent (int): The wanted volume.
+        """
+        self._socket_client.receiver_controller.set_volume(float(percent / 100))  # noqa: 501
+
+    def volume_up(self, delta=0.1):
+        """ Increment volume by 0.1 (or delta) unless it is already maxed.
+        Returns the new volume.
+        """
+        if delta <= 0:
+            raise ValueError(
+                "volume delta must be greater than zero, not {}".format(delta)
+            )
+        return self.set_volume(self.status.volume_level + delta)
+
+    def volume_down(self, delta=0.1):
+        """ Decrement the volume by 0.1 (or delta) unless it is already 0.
+        Returns the new volume.
+        """
+        if delta <= 0:
+            raise ValueError(
+                "volume delta must be greater than zero, not {}".format(delta)
+            )
+        return self.set_volume(self.status.volume_level - delta)
+
+    def mute(self, status=None):
+        """mute the sound, acts as on off.
+
+        Args:
+            status (None, optional): override for on/off
+        """
+        if status is not None:
+            status = status
+        else:
+            status = not status.volume_muted
+
+        self._socket_client.receiver_controller.set_volume_muted(status)
+
+    def show_media(self, media=None, **kwargs):
+        """Show the media on the screen"""
+        msg = media_to_chromecast_command(
+            media,
+            type=TYPE_DETAILS,
+            requestId=self._inc_request(),
+            **kwargs
+        )
+
+        def callback():  # pylint: disable=missing-docstring
+            self._send_cmd(msg, inc_session_id=True, inc=False)
+
+        self.launch(callback)
+
+    def quit_app(self):
+        """Quit the plex app"""
+        self._socket_client.receiver_controller.stop_app()
+
+    @property
+    def status(self):
+        """Get the chromecast playing status.
+
+        Returns:
+            pychromecast.controllers.media.MediaStatus: Slightly modified status with patched
+                                                        method for episode_title.
+        """  # noqa
+
+        @property
```

Comment:
I don't think that this should be defined inside `status` ?

Reply:
Do you want me to add this to MediaStatus or just define in outside the property?